tryouts/py-json.py

```diff
@@ -0,0 +1,12 @@
+import json
```

Comment:
This needs to be removed as well before we release. Please remove tryouts and add it to gitignore.